service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/HumanNameMapper.java

```diff
@@ -0,0 +1,45 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import java.util.stream.Stream;
+import uk.nhs.adaptors.oneoneone.cda.report.util.NodeUtil;
+import uk.nhs.connect.iucds.cda.ucr.PN;
+import org.hl7.fhir.dstu3.model.HumanName;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HumanNameMapper {
+
+    @Autowired
+    PeriodMapper periodMapper;
```

Comment:
See Java coding standards. Let's try to use constructors instead of field injection.
Field should be private